fuel-core/src/service/genesis.rs

```diff
@@ -39,14 +41,6 @@ impl FuelService {
         Ok(())
     }
 
-    /// initialize starting block height if set
-    fn init_block_height(db: &Database, state: &StateConfig) -> Result<()> {
```

Comment:
This isn't the same as the other init fns, it allows the chain config to specify the starting block height of a chain. My thinking was this could be used to re-genesis / fork a network using only the state config (e.g. fork main-net at given height for contract testing)